src/cpp/util/byte_buffer.cc

```diff
@@ -95,4 +102,10 @@ ByteBuffer& ByteBuffer::operator=(const ByteBuffer& buf) {
   return *this;
 }
 
+void ByteBuffer::Swap(ByteBuffer* other) {
+  grpc_byte_buffer* tmp = other->buffer_;
+  other->buffer_ = this->buffer_;
```

Comment:
Yeah, that shouldn't be necessary.  I copied that from the internal CL and didn't notice it before I merged.  Feel free to send me a PR to fix it if it bothers you.